pkg/tfbridge/walk.go

```diff
@@ -114,8 +115,117 @@ func propertyPathToSchemaPathInner(
 	return nil
 }
 
+// Translate a a bridged provider schema path into a Pulumi property path.
+//
+// The function hides the complexity of mapping Terraform names to Pulumi property names,
+// joining Schema with user overrides in [SchemaInfo]s, and accounting for MaxItems=1
+// situations where Pulumi flattens collections to plain values.  and therefore SchemaPath
+// values are longer than the PropertyPath values.
+//
+// [SchemaPathToPropertyPath] may return nil if there is no matching schema found. This
+// may happen when drilling down to values of unknown type, attributes not tracked in
+// schema, or when there is a type mismatch between the path and the schema.
+//
+// ## Element handling
+//
+// [SchemaPath]s can be either attributes or existential elements. For example:
```

Comment:
What does "existential" mean here?

Reply:
I clarified in https://github.com/pulumi/pulumi-terraform-bridge/pull/1594/commits/7cabf06a35f94e56fdf3e76946fc0f8644230de1. In summary:

In a `SchemaPath`, `.Element()` represents the existence of some element access, but not a specific element access. I'm calling that an existential element. This is different from an element index in a `PropertyPath`: I'm calling `[3]` an instantiated element access, since it represents a specific index.